pkg/client/supplier/client.go

```diff
@@ -82,6 +85,16 @@ func (sClient *supplierClient) SubmitProof(
 		return err
 	}
 
+	// TODO_IMPROVE: log details related to what & how much is being proven
+	logger.Info().
+		Fields(map[string]any{
+			"supplier_addr": sClient.signingKeyAddr.String(),
+			"app_addr":      sessionHeader.ApplicationAddress,
+			"session_id":    sessionHeader.SessionId,
+			"service":       sessionHeader.Service.Id,
+		}).
```

Comment:
It looks like the same map literal is being declared twice for logging purposes. Perhaps it would be convenient to assign it to a variable instead (here and line 124)?

Reply:
One of the them is being assigned in the `CreateClaim` function and the other is being assigned in the `SubmitProof` function.

Both are dependant on the input values provided. 

Can you elaborate on what you had in mind?

Reply:
Ignore me :upside_down_face: